packages/SwingSet/tools/paths.js

```diff
@@ -3,9 +3,14 @@ import { resolve } from 'import-meta-resolve';
 
 /**
  * @param {string} specifier
+ * @param {string} base
  * @returns {string} file system absolute path of the package resource specifier
  */
-export const pkgAbsPath = specifier => {
-  const href = resolve(specifier, import.meta.url);
+export const resolvePath = (specifier, base) => {
```

Comment:
Maybe `resolveToPath` would be more clear. The arguments are both module specifiers and a path returns.